### PR TITLE
Fix GIFs

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-####

### CODE REVIEW

File: src/utils/handleNFTImages.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { PixelRatio } from 'react-native';
```
Comment (benisgold) [inline src/utils/handleNFTImages.ts:1]: see comments in this file for details

File: src/handlers/ens.ts [hunk 2]
```diff
@@ -62,7 +62,11 @@ const buildEnsToken = ({
   name: string;
   imageUrl: string;
 }) => {
-  const { imageUrl, lowResUrl } = handleAndSignImages(imageUrl_);
+  const { highResUrl: imageUrl, lowResUrl } = handleNFTImages({
+    originalUrl: imageUrl_,
+    previewUrl: undefined,
+    mimeType: MimeType.SVG,
```
Comment (skylarbarrera) [inline src/handlers/ens.ts:68]: we sure that should be svg?

File: src/utils/handleNFTImages.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import { CardSize } from '@/components/unique-token/CardSize';
+
+const pixelRatio = PixelRatio.get();
+const deviceWidth = deviceUtils.dimensions.width;
+const fullSize = deviceWidth * pixelRatio;
+const MAX_IMAGE_SCALE = 3;
+export const FULL_NFT_IMAGE_SIZE = fullSize * MAX_IMAGE_SCALE;
+
+// mime types provided by SimpleHash
+export enum MimeType {
```
Comment (skylarbarrera) [inline src/utils/handleNFTImages.ts:13]: curious about this cause its missing any regular image types

File: src/utils/handleNFTImages.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  // previewUrl is provided by SimpleHash, expected to be jpeg, png, or gif
+  previewUrl: string | null | undefined;
+  mimeType: string | null | undefined;
+}): { highResUrl: string | null; lowResUrl: string | null } {
+  const url = previewUrl ?? originalUrl;
+  if (!url) {
+    return { highResUrl: null, lowResUrl: null };
+  }
+
+  const isSVG = mimeType === MimeType.SVG;
```
Comment (skylarbarrera) [inline src/utils/handleNFTImages.ts:37]: 👍

**Review by skylarbarrera — APPROVED** [review]
1 smol area of q but otherwise beeeeetttt

**Pull request merged**